core/src/scene/drawRule.h

```diff
@@ -37,11 +38,10 @@ class StyleContext;
 struct DrawRuleData {
 
     std::vector<StyleParam> parameters;
-    std::string name;
-    int id;
+    std::string styleName;
+    int styleId;
```

Comment:
:+1: This also causes a great deal of confusion between js and es implementations so a comment is good for everyone :D.